builder/builder-next/worker/worker_test.go

```diff
@@ -0,0 +1,94 @@
+package worker
+
+import (
+	"testing"
+
+	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
+	"gotest.tools/v3/assert"
+	is "gotest.tools/v3/assert/cmp"
+)
+
+func Test_Worker_Platforms(t *testing.T) {
```

Comment:
Micro-nit; we usually don't use underscores in our test-names. I know it's a common convention elsewhere (and nothing bad on that), but perhaps align with what we do in this repo for consistency;

```suggestion
func TestWorkerPlatforms(t *testing.T) {
```


(Happy with the table test; indeed looks a lot better!)

Reply:
No worries, done.